scene/gui/foldable_container.cpp

```diff
@@ -0,0 +1,412 @@
+/**************************************************************************/
+/*  foldable_container.cpp                                                */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
```

Comment:
Might want to add Blazium Copyright in here somewhere

Reply:
We will not pass the tests if we do it right now, because we didn't add this line anywhere inside the dev branch yet, also adding it will make it harder to compare the project to any other godot project at the moment to make it easier to compare the changes, ie. diff folders plugin for VSCode.

Reply:
no need for blazium copyright as of right now